src/main/java/app/gpx_animator/core/configuration/Configuration.java

```diff
@@ -157,14 +162,14 @@ private Configuration(
             final Long totalTime, final float backgroundMapVisibility, final String tmsUrlTemplate, final String tmsApiKey, final String tmsUserAgent,
             final boolean skipIdle, final Color backgroundColor, final File backgroundImage, final Color flashbackColor,
             final Long flashbackDuration, final boolean preDrawTrack, final Long keepFirstFrame, final Long keepLastFrame, final File output,
-            final VideoCodec videoCodec, final String attribution, final String information, final SpeedUnit speedUnit, final Font font,
-            final Double markerSize, final Font waypointFont, final Double waypointSize, final Double minLon, final Double maxLon,
+            final VideoCodec videoCodec, final MusicCodec musicCodec, final String attribution, final String information, final SpeedUnit speedUnit,
+            final Font font, final Double markerSize, final Font waypointFont, final Double waypointSize, final Double minLon, final Double maxLon,
             final Double minLat, final Double maxLat, final File logo, final Position logoPosition, final int logoMargin,
             final Position attributionPosition, final int attributionMargin,
             final Position informationPosition, final int informationMargin,
             final Position commentPosition, final int commentMargin,
-            final File photoDirectory, final long photoFreezeFrameTime, final Long photoTime, final Long photoAnimationDuration,
-            final boolean preview, final Long previewLength,
+            final File photoDirectory, final File inputMusic, final long photoFreezeFrameTime, final Long photoTime,
```

Comment:
You added the parameter `File inputMusic` right between the parameters for the photos. There are already so many parameters on this constructor - I would like to keep similar parameters together, like not to break up the photo parameters. What about just switching the `photoDirectory` with the `inputMusic`? That would keep the photo parameters together. Or move the `inputMusic` right after the `musicCodec` to keep the music parameters togethers. I'm fine with both.

Reply:
ok fixed